### ISSUE TITLE
EPIC 214155925 (Ezra)

### ISSUE THREAD

**Issue by George-Hookway** [George-Hookway/PBjammy#100]
![214155925 - 11 - SG - Echelle](https://github.com/user-attachments/assets/85c18adb-78fe-4a5c-b76f-d312ee8878cb)
![214155925 - 11 - SG - Mode ID](https://github.com/user-attachments/assets/b7bf9e30-a6b7-4d3d-8b02-191f86169e5f)
![214155925 - 11 - SG - Peakbagged Echelle](https://github.com/user-attachments/assets/9fd57605-b2af-43f0-848f-8031aaeb32f2)
![214155925 - 11 - SG - Peakbagging](https://github.com/user-attachments/assets/a97da161-86d4-4738-a30d-0e6ebb62657f)

@nielsenmb @grd349

**Comment by grd349** [comment]
I think it has missed a mixed mode at low frequency. We should discuss with Martin.

**Comment by nielsenmb** [comment]
Yeah, the period spacing is wrong on this one. I think ng=1 should be the one causing the first avoided crossing at 750muHz
